src/layouts/Header.tsx

```diff
@@ -84,6 +84,8 @@ class Header extends React.Component<IProps, IStateProps> {
     enableWalletProvider({
       suppressNotifyOnSuccess: true,
       showNotification: this.props.showNotification,
+    }).catch((err) => {
```

Comment:
`enableWalletProvider` never throws an exception.  Instead it shows a notification and returns false (so we are already feeding back to the user)